custom/icds_reports/migrations/0100_add_supervisor_id.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-02-21 09:32
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('icds_reports', '0099_service_delivery_report'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='aggregatebirthpreparednesforms',
+            name='supervisor_id',
+            field=models.CharField(max_length=40, null=True),
```

Comment:
these must all be `Text` to match the UCRs so that we get co-location with citusdb